src/package_manager/pip.ts

```diff
@@ -40,6 +40,7 @@ const pip3Packages: string[] = [
 	"mock",
 	"mypy",
 	"nose",
+	"numpy",
```

Comment:
adding numpy here

Reply:
One thing I don't understand is how the ROS2 Wiki doesn't have this listed as a dependency to install, so what else would be pulling it in? Or maybe it's just missing from their steps?